bin/oneoff/nces_data/import_school_stats_by_years_2020_2021_pss

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+CDO.log = Logger.new(STDOUT)
+
+SURVEY_YEAR = '2019-2020'.freeze
```

Comment:
should this be "2020-2021"?

Reply:
No- it's actually the 2019-2020 data. I'm not sure why we didn't add it this year, or if it was only just released. I originally thought this too until I opened up the CSV files and saw that nearly every column was labeled with the years😂

Reply:
Okay sounds good! (Also I don't know why github was posting my comments multiple times...)

Reply:
I just saw your comment that you looked at the csv too:)